src/components/Pagination.tsx

```diff
@@ -1,6 +1,9 @@
 import { Button, Icon, Input, Select } from "@canonical/react-components";
-import React, { FC } from "react";
+import React, { FC, useState } from "react";
 import { paginationOptions } from "util/pagination";
+import useEventListener from "@use-it/event-listener";
+
+const figureSmallScreen = () => window.innerWidth < 850;
```

Comment:
As you already noticed, I don't really like seeing these values hardcoded here and there. It's not very important, but if we can assign this to a `const` in a central place and reference it from there, that would make me happier.

Also, why is it 850? I still experience the 2 lines between 850 and 863 width:

![image](https://user-images.githubusercontent.com/56583786/230156710-11626575-02cb-4cb2-89c9-ab4145c568ea.png)

Reply:
I am no longer able to reproduce this issue.

Reply:
> Also, why is it 850? I still experience the 2 lines between 850 and 863 width:

This was wired, it is also dependent on the side panel being open or not. I figured it is way safer to measure the available space for the text directly and this should resolve it.

Reply:
Ok, so to summarise, we're happy with it taking 2 lines with the side panel open? Because this is what I currently get at 1240 width as soon as I open a side panel:

![image](https://user-images.githubusercontent.com/56583786/230324380-aa6dfe7d-1782-4713-9c00-4fd3d18aee54.png)

If this is expected/intended, just let me know.

Reply:
I believe this is not expected. I have just checked, it does not behave that way for me. It goes straight to "50 out of 59"

Reply:
![Screenshot 2023-04-06 at 12 28 26](https://user-images.githubusercontent.com/114913788/230335345-9ae9621f-6dad-45db-aa5f-4fac2dfbf3be.png)

Reply:
This is what I experience with Chrome version 112.0.5615.49 on Ubuntu:

![Peek 2023-04-06 11-44](https://user-images.githubusercontent.com/56583786/230340042-f8ae2e20-9201-4849-8eff-4a33f2442818.gif)

Reply:
Oh right, the pagination does not get updated on sidepanel open. I think we should throw a resize event when we open or close the aside. I'll add that in.